vrk-kpa/suomidigi
# SUOPA-934 Infinite scroll

To test: 
- `make fresh`
- Check that the site works and own profile page shows "load more" button on first page load.
